src/commitlint/cli.py

```diff
@@ -57,28 +58,43 @@ def get_args() -> argparse.Namespace:
     # --to-hash is optional
     parser.add_argument("--to-hash", type=str, help="To commit hash", default="HEAD")
 
+    # feature options
+    parser.add_argument(
+        "--quick", action="store_true", help="Check the commit message quickly."
```

Comment:
suggestion: Semantically speaking `short` and `quick` do not mean the same thing. Wouldn't changing this to `short` make more sense?

Reply:
Or alternatively changing all the occurrences of `short` in the codebase to `quick`?

Reply:
I initially used the word 'quick' as the lint process is done quickly with a single pattern check and failing fast. The word 'short' is only used to display the error message in short (_show_errors).

We can replace all occurrences of 'short' with 'quick', as 'short' is always True when 'quick' is True. What do you suggest?

Reply:
Let's go with `short` as this would be clearer. (Also, to the end user who might not be interested in the implementation.)

Reply:
Replacing `quick` with `short`, the CLI usage will be like this: 
```bash
$ commitlint --short "feat: please my commit message"
```
I don't think this makes sense.

Reply:
`swift` or `partial` are better alternatives as well

Reply:
After our discussion with @sugat009, we have decided to add `--skip-detail` as the flag.
```bash
$ commitlint --skip-detail "feat: my commit message"
```
cc: @subashcs

Reply:
@sugat009 , I have updated the PR.